sdk/loadtestservice/ci.yml

```diff
@@ -0,0 +1,32 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
```

Comment:
rename this file to ci.mgmt.yml, and change the config as https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/edgeorder/ci.mgmt.yml

Reply:
@ArthurMa1978 we would be having our data plane SDK as well (work in progress), under the same directory. 
We would require the CI jobs to kickoff for both mgmt as well as data plane, and hence we had named it ci.yml as per [doc #11](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process).
Please let us know if we still need to update the ci file name particular to mgmt.

Reply:
We have split the management plane and data plane pipeline, so please follow Arthur's comment to update this file.

Reply:
updated the file